Lombiq.Tests.UI/Services/WebDriverFactory.cs

```diff
@@ -173,20 +171,10 @@ private static TDriver CreateDriver<TDriver>(IDriverConfig driverConfig, Func<TD
 
         try
         {
-            // Firefox: The FirefoxConfig.GetMatchingBrowserVersion() resolves the browser version but not the
-            // geckodriver version.
-            version = driverConfig is FirefoxConfig
-                ? driverConfig.GetLatestVersion()
-                : driverConfig.GetMatchingBrowserVersion();
-
             // While SetUpDriver() does locking and caches the driver it's faster not to do any of that if the setup was
             // already done. For 100 such calls it's around 16s vs <100ms. The Lazy<T> trick taken from:
             // https://stackoverflow.com/a/31637510/220230
-            _ = _driverSetups.GetOrAdd(driverConfig.GetName(), _ => new Lazy<bool>(() =>
-            {
-                new DriverManager().SetUpDriver(driverConfig, version);
-                return true;
-            })).Value;
+            version = _driverSetups.GetOrAdd(browserName, name => new Lazy<string>(() => DriverSetup.AutoSetUp(name).BrowserName)).Value;
```

Comment:
Is this kind of caching still needed?

Reply:
[I did some measurements](https://github.com/Lombiq/UI-Testing-Toolbox/pull/303/commits/6e96e930da403d5ddb3369227f2a62b11a0da33b) and I don't think it's needed any more.

Reply:
Great then!